lib/api/build.rs

```diff
@@ -7,6 +7,8 @@ fn main() -> std::io::Result<()> {
         // so by extending the builder. This is ugly, but better than manually implementing
         // `Validation` for all these types and seems to be the best approach. The line below
         // configures all attributes.
+        .extern_path(".google.protobuf.Empty", "()")
+        .compile_well_known_types(true)
```

Comment:
Yes, and also to generate the other well-known types. Let me summarize:

Multiple "protobuf well-known types" (from the "google/protobuf" namespace) are used in the grpc messages. By default, these come from `prost-types`, but those definitions don't implement `serde::Serialize`, which in turn causes deriving `Serialize` and in turn also `Validate` to fail for any types depending on them. So, we make `prost` generate them from their definition instead, so we can add `#[derive(Serialize)]` to them. `prost-types` just aliases some types instead of using wrappers, because that's more ergonomic. The only one of these wrapper types that is used in qdrant is `Empty`, so I just aliased that here. The other types come from `google.protobuf.rs` that's generated by `compile_well_known_types(true)`.